sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/AggregatorFactory.java

```diff
@@ -77,6 +77,18 @@ static AggregatorFactory minMaxSumCount() {
     return MinMaxSumCountAggregatorFactory.INSTANCE;
   }
 
+  /**
+   * Returns an {@code AggregatorFactory} that calculates an approximation of the distribution of
+   * the measurements taken.
+   *
+   * @param temporality configures what temporality to be produced for the Histogram metrics.
+   * @param boundaries configures the fixed bucket boundaries.
+   * @return an {@code AggregationFactory} that calculates histogram of recorded measurements.
+   */
+  static AggregatorFactory histogram(double[] boundaries, AggregationTemporality temporality) {
```

Comment:
Was thinking about this API, probably better to accept `List<Double>` here, and internally use the same array. Idea being that List is a bit more natural as an input param than double[]. What do you think? @jkwatson what do you think?

Reply:
No strong opinion on this one. Either way, we should absolutely make a defensive copy of the data. Certainly I have no objection to using a List<Double> for the input boundaries here.

Reply:
We do the defense copy already :)